products/ha_memgraph/v0.15.0/how_to_guides/configure-and-run-ha-memgraph.md

```diff
@@ -0,0 +1,115 @@
+
+We also need to specify two `.json` files. One of them (`raft_config_file`)
+defines some constants which are internal to the Raft protocol. The other one
+(`coordination_config_file`) contains coordination info such as network
+endpoints and ports.
+
+Finally, we need to specify the location on disk where persistent data should
+be stored. In order to do this, we use the `--durability-directory` flag.
+
+Therefore, we can run the `memgraph_ha` binary on the first ( ID=1`) server by
+issuing the following command:
+
+```plaintext
+  ./memgraph_ha --server_id 1 \
+                --coordination_config_file="coordination.json" \
+                --raft_config_file="raft.json" \
+                --durability_directory=dur$id --port 7001
+```
+
+By analogy, let's suppose that the second server will be run with `--server_id`
+equal to `2`, `durability_directory` equal to `dur2` and `port` equal to `7002`.
+The same analogy follows for the third server.
+
+The assumed contents of the `coordination.json` file are:
+
+```plaintext
+[
+  [1, "127.0.0.1", 7001],
+  [2, "127.0.0.1", 7002],
+  [3, "127.0.0.1", 7003]
+]
+```
+
+Here, each line corresponds to coordination of one server. The first entry is
+that server's ID, the second is its IP address and the third is the port it
+listens to.
+
+The assumed contents of the `raft.json` file are:
+
+```plaintext
+{
+  "election_timeout_min": 150,
+  "election_timeout_max": 300,
+  "heartbeat_interval": 100,
+  "replication_timeout": 10000,
+  "log_size_snapshot_threshold": -1
+}
+```
+
+The meaning behind each entry is demistified in the following table:
+
+Flag                          | Description
+------------------------------|------------
+`election_timeout_min`        | Lower bound for the randomly sampled reelection timer given in milliseconds
+`election_timeout_max`        | Upper bound for the randomly sampled reelection timer given in milliseconds
+`heartbeat_interval`          | Time interval between consecutive hearbeats given in milliseconds
+`replication_timeout`         | Time interval allowed for data replication given in milliseconds
+`log_size_snapshot_threshold` | Allowed number of entries in Raft log before its compaction
+
+### Querying HA Memgraph Using the HA Client
+
+[//]: # TODO when HA Client is implemented
+
+### Querying HA Memgraph Without the HA Client
+
+If you wish to use a third-party client such as `neo4j-client` you should be
+aware that the client should only communicate with the leader of the cluster.
+
+Suppose we have a cluster of 3 machines and have observed the following message
+on the third machine soon after startup:
+
+```plaintext
+I0327 13:46:29.906607 8200 :590] Server 3: Transitioned to LEADER (Term: 6)
+```
+
+This lets you know that the third machine was successfully elected as a leader
+and should be ready to process queries. Since we know that this machine lives
+on `localhost` and listens on `7003`, we can connect to it using the
```

Comment:
We don't know that this machine lives on localhost. We know the leader is the `thrid` machine which can have any IP address. I would put the same IP as in the above config file.